dirty_cat/test/test_super_vectorizer.py

```diff
@@ -18,6 +18,17 @@ def check_same_transformers(expected_transformers: dict, actual_transformers: li
     actual_transformers_dict = dict([(name, cols) for name, trans, cols in actual_transformers])
     assert actual_transformers_dict == expected_transformers
 
+def type_equality(expected_type, actual_type):
+    """
+    Checks that the expected type is equal to the actual type, assuming object and str types
+    are equivalent (considered as categorical by the supervectorizer).
+    """
+    if (isinstance(expected_type , object) or  isinstance(expected_type, str))\
```

Comment:
I don't need to protect a line return if it is in the middle of enclosing parenthesis.